habushu-maven-plugin/src/test/java/org/technologybrewery/habushu/DeployPhaseSteps.java

```diff
@@ -26,6 +26,16 @@ public void a_custom_push_dev_repository_url_path_of(String customPushDevReposit
         deployMojo.devRepositoryUrlUploadSuffix = customPushDevRepositoryUrlPath;
     }
 
+    @Given("an null URL")
+    public void an_null_url() {
+        deployMojo.devRepositoryUrl = null;
+    }
+
+    @Given("an empty URL")
```

Comment:
Does this need to be set to an empty URL rather than a null?

Reply:
Indeed it does - ✅ FIXED